src/app/state/movies/movies.facade.ts

```diff
@@ -8,6 +8,7 @@ import * as moviesSelectors from './movies.selectors';
 @Injectable()
 export class MovieFacade {
   public movies$ = this.store.select(moviesSelectors.selectMovie);
+  public loading$ = this.store.select(moviesSelectors.selectLoading);
```

Comment:
ok tutaj ściągnąłeś ode mnie trochę widzę ten loading, ale pomyśl też kopiując :D no bo obczaj jak masz loading na całym state movies, to musisz chować cały array filmów dodając nowy film. Tzn dodajesz film, zmieniasz movies.loading na true i wtedy na widoku chowasz na chwile wszystkie filmy, a pododaniu filmu przez addMovieSuccess zmieniasz flagę na false i widac wszystkie filmy.
Ok, możemy tak zrobić, ale zaczęliśmy kminić już inaczej, tzn że nie ma flagi movies.loading tylko każdy film ma na sobie loading i każdy film osobno może się ładowac bez chowania innych. Możemy to zmienić, ale tutaj robisz dwie wersje na raz, zdecyduj się która Ci bardziej pasuje i alob wyjeba loading ze stanu filmów albo wyjeb z Movie. Jestem otwarty na obie opcje, może i łatwiej będzie robić loading na całym stanie